From b23ba97c3be184f5949e60bfc7cf050a5d317fca Mon Sep 17 00:00:00 2001
From: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Date: Mon, 9 Apr 2018 18:44:07 +0700
Subject: [PATCH 477/628] ARM: DTS: r8a7742: Change to use clock-indices

commit 8e33f91a0b84 ("clk: shmobile: clk-mstp: change to using clock-indices")
    "With the addition of clock-indices, we need to change the renesas
    clock implementation to use these instead of the local definition
    of "renesas,clock-indices"'

Signed-off-by: Binh Nguyen <binh.nguyen.uw@rvc.renesas.com>
Signed-off-by: vietn <vietn@fsoft.com.vn>
---
 arch/arm/boot/dts/r8a7742.dtsi | 20 ++++++++++----------
 1 file changed, 10 insertions(+), 10 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742.dtsi b/arch/arm/boot/dts/r8a7742.dtsi
index e87983c..1853cf4 100644
--- a/arch/arm/boot/dts/r8a7742.dtsi
+++ b/arch/arm/boot/dts/r8a7742.dtsi
@@ -977,7 +977,7 @@
 			reg = <0 0xe6150130 0 4>, <0 0xe6150030 0 4>;
 			clocks = <&mp_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <R8A7742_CLK_MSIOF0>;
+			clock-indices = <R8A7742_CLK_MSIOF0>;
 			clock-output-names = "msiof0";
 		};
 		mstp1_clks: mstp1_clks@e6150134 {
@@ -990,7 +990,7 @@
 				 <&p_clk>, <&rclk_clk>, <&cp_clk>, <&zs_clk>,
 				 <&zs_clk>, <&zs_clk>, <&zs_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_VCP1 R8A7742_CLK_VCP0
 				R8A7742_CLK_VPC1 R8A7742_CLK_VPC0
 				R8A7742_CLK_SSP1 R8A7742_CLK_TMU1
@@ -1016,7 +1016,7 @@
 				<&mp_clk>, <&mp_clk>, <&mp_clk>, <&mp_clk>,
 				<&mp_clk>, <&zs_clk>, <&zs_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_SCIFA2 R8A7742_CLK_SCIFA1
 				R8A7742_CLK_SCIFA0 R8A7742_CLK_MSIOF2
 				R8A7742_CLK_SCIFB0 R8A7742_CLK_SCIFB1
@@ -1039,7 +1039,7 @@
 				 <&hp_clk>, <&mp_clk>, <&hp_clk>, <&mp_clk>,
 				 <&rclk_clk>, <&hp_clk>, <&hp_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_IIC2 R8A7742_CLK_TPU0
 				R8A7742_CLK_MMCIF1 R8A7742_CLK_SDHI3
 				R8A7742_CLK_SDHI2 R8A7742_CLK_SDHI1
@@ -1061,7 +1061,7 @@
 			reg = <0 0xe6150140 0 4>, <0 0xe615004c 0 4>;
 			clocks = <&cp_clk>, <&rclk_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 					R8A7742_CLK_IRQC R8A7742_CLK_RWDT>;
 			clock-output-names = "irqc", "rwdt";
 		};
@@ -1071,7 +1071,7 @@
 			reg = <0 0xe6150144 0 4>, <0 0xe615003c 0 4>;
 			clocks = <&zs_clk>, <&zs_clk>, <&extal_clk>, <&p_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_AUDIO_DMAC0 R8A7742_CLK_AUDIO_DMAC1
 				R8A7742_CLK_THERMAL R8A7742_CLK_PWM>;
 			clock-output-names =
@@ -1085,7 +1085,7 @@
 			<&p_clk>, <&p_clk>, <&zx_clk>, <&zx_clk>, <&zx_clk>,
 			<&zx_clk>, <&zx_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_EHCI R8A7742_CLK_HSUSB
 				R8A7742_CLK_HSCIF1 R8A7742_CLK_HSCIF0
 				R8A7742_CLK_SCIF1 R8A7742_CLK_SCIF0
@@ -1104,7 +1104,7 @@
 			clocks = <&zg_clk>, <&zg_clk>, <&zg_clk>, <&zg_clk>,
 				<&p_clk>, <&hp_clk>, <&zs_clk>, <&zs_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_VIN3 R8A7742_CLK_VIN2
 				R8A7742_CLK_VIN1 R8A7742_CLK_VIN0
 				R8A7742_CLK_ETHER R8A7742_CLK_ETHERAVB
@@ -1124,7 +1124,7 @@
 				 <&cpg_clocks R8A7742_CLK_QSPI>, <&cp_clk>,
 				 <&hp_clk>, <&hp_clk>, <&hp_clk>, <&hp_clk>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_GPIO5 R8A7742_CLK_GPIO4
 				R8A7742_CLK_GPIO3 R8A7742_CLK_GPIO2
 				R8A7742_CLK_GPIO1 R8A7742_CLK_GPIO0
@@ -1160,7 +1160,7 @@
 				<&mstp10_clks R8A7742_CLK_SCU_ALL>,
 				<&mstp10_clks R8A7742_CLK_SCU_ALL>;
 			#clock-cells = <1>;
-			renesas,clock-indices = <
+			clock-indices = <
 				R8A7742_CLK_SCU_ALL R8A7742_CLK_SSI_ALL
 				R8A7742_CLK_SSI9 R8A7742_CLK_SSI8
 				R8A7742_CLK_SSI7 R8A7742_CLK_SSI6
-- 
2.7.4

